File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.js
function solution(s) {
    const words = s.split(" ");
    
    for(let i=0; i<words.length; i++){
        for(let j=0; j<words[i].length; j++){
            words[i] = words[i][0].toUpperCase() + words[i].slice(1).toLowerCase();
        }
    }
    
    return words.join(" ");
    
}

File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.py
def solution(s):
    arr = s.split(' ')
    arr = [a.lower() for a in arr]
    
    for i in range(len(arr)):
        if len(arr[i]) > 0:
            arr[i] = arr[i][0].upper() + arr[i][1:]
        
    print(arr)
    
    return ' '.join(arr)